content/docs/command-reference/fetch.md

```diff
@@ -1,7 +1,8 @@
 # fetch
 
-Get tracked files or directories from
-[remote storage](/doc/command-reference/remote) into the <abbr>cache</abbr>.
+Download <abbr>cache</abbr> files or directories from
+[remote storage](/doc/command-reference/remote), based on the current `dvc.yaml`
```

Comment:
dvc.lock ... also I would try to avoid reference to such level of details here ... also, why do we change this, btw?

Reply:
also, it's not exactly true - there are options to take into account history

Reply:
Good call. I moved the detail about dvc.yaml/lock files from the first line down to the description.

> why do we change this

Most of these regular updates come from recent(ish) conversations and discussions in different places e.g. other PRs. They tend to add up in my Git stash and I lose the exact references, sorry.